scripts/chain_configs/localterra.json

```diff
@@ -19,7 +19,18 @@
       "owner": "",
       "expires_in": 604800
     },
-    "allocations": []
+    "allocations": [
+      "terra1x46rqay4d3cssq8gxxvqz8xt6nwlz4td20k38v",
+      {
+        "amount": "1000000000",
```

Comment:
[here](https://github.com/astroport-fi/astroport-governance/blob/bec6609364cc0ac7f46092c7bf017f1574711b42/scripts/deploy.ts#L183) we will get an error. We should set max_allocations_amount to sum of allocations.